Pod/Classes/SEGSwrveIntegration.m

```diff
@@ -14,14 +14,26 @@ - (instancetype)initWithSettings:(NSDictionary *)settings
 
 - (void)identify:(SEGIdentifyPayload *)payload
 {
+    NSMutableDictionary *traitsCopy = [payload.traits mutableCopy];
+    if ([[payload.traits allKeys] containsObject:@"swrve_external_id"]) {
+        NSString *external_id = [payload.traits objectForKey:@"swrve_external_id"];
+        [SwrveSDK identify:external_id onSuccess:^(NSString* status, NSString *swrveUserId) {
+            SEGLog(@"Successfully identified swrve_user_id %@ with external_id %@", swrveUserId, external_id);
```

Comment:
In Swrve documentation https://docs.swrve.com/developer-documentation/integration/ios/#User_identity there are recommendation:

```If the call fails([SwrveSDK identify:]), then the user will not be correctly linked on the Swrve backend system, which may affect reporting and audiences where the user is logging in on multiple devices. We recommend not queuing or sending events until the identify call completes.```

SwrveSDK identify call is an asynchronous call, whether Segment identify is synchronous. 
So all events that will be tracked while SwrveSDK identify is processing, will be attached to the wrong/previous user/device. 

I don't think that a lot could be done here. As an option is to Queue all Segment events and wait until SwrveSDK identify call will be completed. But the complexity of this solution is too risky.